db/migrate/20160912000001_set_default_run_only_when_new_commit.rb

```diff
@@ -0,0 +1,5 @@
+class SetDefaultRunOnlyWhenNewCommit < ActiveRecord::Migration
+  def change
+    change_column :crons, :run_only_when_new_commit, :boolean, :default => false
```

Comment:
Yes because this column originally has `run_only_when_new_commit boolean DEFAULT true NOT NULL` since this was the old setting